### ISSUE TITLE
Integration tests app does not load CMS images

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15185]
## Description

CMS images are 404 on https://test.bedrock.nonprod.webservices.mozgcp.net/en-US/about/leadership/ since they try to load from `storage.googleapis.com`

**Comment by stevejalim** [comment]
@alexgibson given that test uses the sqlite-exported Dev DB (which comes out every 12 hours) and doesn't upload any data, we could make bedrock-test use the same GCS bucket spec as dev and load assets from there. I don't think we want a CDNed MEDIA_URL for bedrock-test, so that should be fine

**Comment by stevejalim** [comment]
@alexgibson Is this blocking anything right now, or (naturally) just a good thing to sort out

**Comment by stevejalim** [comment]
Waiting on SRE feedback as I can't seem to add the right Secrets to configure bedrock-test the way I think we could

**Comment by alexgibson** [comment]
>  Is this blocking anything right now, or (naturally) just a good thing to sort out

Nope, just something that's a nice to have :)
